docker/cortx-deploy/build.sh

```diff
@@ -63,12 +65,24 @@ echo "Building $SERVICE image from $BUILD_URL"
 sleep 5
 
 function get_git_hash {
-sed -i '/KERNEL/d' RELEASE.INFO
-for component in cortx-py-utils cortx-s3server cortx-motr cortx-hare cortx-provisioner
-do
-    echo $component:"$(awk -F['_'] '/'$component'-'$VERSION'/ { print $2 }' RELEASE.INFO | cut -d. -f1 | sed 's/git//g')",
-done
-echo cortx-csm_agent:"$(awk -F['_'] '/cortx-csm_agent-'$VERSION'/ { print $3 }' RELEASE.INFO | cut -d. -f1)",
+        sed -i '/KERNEL/d' RELEASE.INFO
+        if [ "$SERVICE" == "cortx-data" ]; then 
+                COMPONENTS="cortx-hare cortx-motr cortx-py-utils cortx-provisioner"
```

Comment:
Should we get a component list from service specific  `cortx-componenet-rpms.txt` to keep a single point of contact for CORTX component update.

Reply:
Updated the logic as per ask and tested - http://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/nikhildev/job/cortx-granular-docker-image/34/